ui/page/stats.page.ts

```diff
@@ -109,16 +112,41 @@ export const StatsPage: Page<StatsModel> = (model: StatsModel) => {
                         ListItem(`${model.totalNumberOfVotes} votes in total`),
                         ListItem(`${model.percentageOfPositiveVotes}% positive`),
                     ),
+                    Heading3(`Cache stats:`),
+                    UnorderedList(
+                        ListItem(`${model.cacheStats.cacheHits} hits, ${model.cacheStats.cacheMisses} misses`),
+                    ),
                 ),
             ),
             Heading3("Actions: "),
             Row(
-
-
                 Form("/admin/css-cache/clear",
                     SubmitButton("Clear css cache")
                 ),
-
+            ),
+            Heading3("App settings: "),
+            Row(
+                UnorderedList(
+                    ...Object.keys(model.appSettings).map((appSetting: keyof AppSettings) => 
+                        ListItem(
+                            Form<AppSettings>("/admin/app-settings",
+                                HiddenInput({ 
+                                    name: "areOptimizationHacksEnabled",
+                                    value: appSetting === "areOptimizationHacksEnabled" 
+                                        ? !model.appSettings.areOptimizationHacksEnabled
+                                        : model.appSettings.areOptimizationHacksEnabled
+                                }),
+                                HiddenInput({ 
+                                    name: "isCacheEnabled",
+                                    value: appSetting === "isCacheEnabled" 
+                                        ? !model.appSettings.isCacheEnabled
+                                        : model.appSettings.isCacheEnabled
+                                }),
+                                SubmitButton(`${appSetting} - ${model.appSettings[appSetting]}`)
```

Comment:
Could we print 'enable' or 'disable' instead of the current state in the button? I find this a bit confusing :sweat_smile:

Reply:
Fixed. 😅